app.ts

```diff
@@ -0,0 +1,163 @@
+﻿const host = "http://localhost:2050";
+let grid: Grid;
+const initRecord = 0;
+const rowLimit = 22;
+
+class Grid {
+    from: number;
+    to: number;
+    headers: string[] = [];
+
+    $row = $("<div />", {
+        class: 'row'
+    });
+
+    $header = $("<div />", {
+        class: 'header'
+    });
+
+    $column = $("<div />", {
+        class: 'column'
+    });
+
+    //sets initial grid values.
+    constructor(from: number, to: number) {
+        this.from = from;
+        this.to = to;
+
+        $.getJSON(host + "/columns", (data) => { })
```

Comment:
I suppose the host should also be passed into this class to decouple it from the global variables